### PR TITLE
perk svelte finished

### PR DESCRIPTION
**Pull request by Waluiyeegi**

### CODE REVIEW

File: client/src/components/Perk.svelte [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<script>
+    import { onMount } from 'svelte';
+
+    export let buttonText = 'View Perk';
+
+    let text = 'Perk Loading...';
+
+    onMount(async () => {
+        const response = await fetch('/api/dynamic-text');
```
Comment (Triguy12) [inline client/src/components/Perk.svelte:9]: Reminder to change fetch location once the PerkController is set up. It should fetch the specific Perk from the PerkRepository

File: server/src/main/java/User.java [hunk 2]
```diff
@@ -8,6 +13,7 @@ public class User {
     private String userName;
     private String password;
     private List<Perk> perksPosted;
+
```
Comment (Triguy12) [inline server/src/main/java/User.java:16]: try to avoid unnecessary changes, also the imports aren't being used right now.

**Review by Triguy12 — DISMISSED** [review]
I looked at a preview of the html, the perk component looks really good! The script section will need to be changed at later iteration.

**Review by Zhangjason308 — APPROVED** [review]
Looks good! Try not to meddle with other features to avoid merge conflicts

**Review by Triguy12 — CHANGES_REQUESTED** [review]
Change the script section to contain perk object. 
Remove onMount section and add it to the svelte file that displays the actual perk object after selecting 'view perk'

**Review by Triguy12 — APPROVED** [review]
Changes look good!
